src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -628,7 +628,7 @@ class KotlinCompilationTests {
 	fun `detects the plugin provided for compilation via pluginClasspaths property`() {
 		val result = defaultCompilerConfig().apply {
 			sources = listOf(SourceFile.kotlin("kSource.kt", "class KSource"))
-			pluginClasspaths = listOf(classpathOf("kotlin-scripting-compiler-1.3.50"))
+			pluginClasspaths = listOf(classpathOf("kotlin-scripting-compiler-1.3.60"))
```

Comment:
Nit: would be nice if this could somehow share a version with the build system

Reply:
This is only used as an "example plugin" to test `pluginClasspath`. I suppose we could also define our own plugin and try to get the classpath of that, but it would be more complicated.